gdk4/src/auto/enums.rs

```diff
@@ -792,6 +792,8 @@ pub enum EventType {
     #[cfg_attr(docsrs, doc(cfg(feature = "v4_8")))]
     #[doc(alias = "GDK_TOUCHPAD_HOLD")]
     TouchpadHold,
+    #[doc(alias = "GDK_EVENT_LAST")]
```

Comment:
those shouldn't be generated, you should replace alias = true with ignore

Reply:
Fixed.